auto_submit/lib/request_handling/pubsub.dart

```diff
@@ -35,4 +36,30 @@ class PubSub {
     final PublishResponse response = await pubsubApi.projects.topics.publish(request, _topic);
     log.info('pubsub response messageId=${response.messageIds}');
   }
+
+  /// Pulls messages from the server.
+  Future<PullResponse> pull(int maxMessages, String subscription) async {
+    final Client httpClient = await clientViaApplicationDefaultCredentials(scopes: <String>[
+      PubsubApi.pubsubScope,
+    ]);
+    final PubsubApi pubsubApi = PubsubApi(httpClient);
+    PullRequest pullRequest = PullRequest(maxMessages: maxMessages);
```

Comment:
This is confusing as we're talking about the PubSub PullRequest. We can make it clearer by prefixing the import.

`import 'package:googleapis/pubsub/v1.dart' as pubsub;`

Reply:
Sure, added the prefix.